### DISCUSSION TITLE
Charts of monitored variables in dynamics mode

### DISCUSSION THREAD

**Discussion by svidille** [DanWBR/dwsim#584] [General]
Dear Dan and everybody else,

In the attached file there is a chart that represents some monitored variables in a specific integrator. My questions are:

- Why do I have time = 3e8 seconds if I run the simulation for one minute only with step size of 0.5 s? Is it possible to reset the time?

- Is it possible to configure those trends (escale, axis, color, title, etc)? I found a change appearance option but I could not see any changes (only in the size of the chart).

- Is it possible to copy the variables from one integrator to another? 

- Is it possible to import/export monitored variables?

- Is it possible to drag and drop variables into the charts?

[Coolers_heaters_test - copia.zip](https://github.com/DanWBR/dwsim/files/14008835/Coolers_heaters_test.-.copia.zip)

Thanks once more,
Best wishes,
Simone

**Comment by DanWBR** [comment]
Hi @svidille, may I ask why so many questions? Are you validating DWSIM against something, is it for work or just for fun?

**Comment by svidille** [comment]
Hi, Dan

I am validating DWSIM Dynamics to see the possibilities for future applications. 

I know in the past there was a note saying that Dynamics was only for academic purposes and it was not validated. However I don´t see it in the actual version, so I suppose now it is possible to use it freely?

Sorry for so many questions. As I commented, maybe I am suggested to the way I use to model in other simulators.

Please, if this is not the correct way to address my questions in this specific case, let me know which path I should follow.


Thanks for your attention,
Best wishes,
Simone
